package.json

```diff
@@ -104,7 +104,7 @@
     "ember-bunsen-core": "0.24.0",
     "ember-cli-babel": "^5.1.8",
     "ember-cli-htmlbars": "^1.0.3",
-    "ember-frost-core": "1.12.0",
+    "ember-frost-core": "1.13.0",
```

Comment:
It actually has a caret now, I fixed it earlier today. Just want to make sure the minimum version isn't changed here.